bin/addEntrypoint.bash

```diff
@@ -125,6 +130,11 @@ select ENTRY_POINT_PARENT in "${MENUS[@]}"; do
 done
 echo ""
 
+# Create the feature branch
```

Comment:
I think the feature branch should still be created after the confirmation below.